yellowbrick/features/missing.py

```diff
@@ -0,0 +1,129 @@
+# yellowbrick.features.missing
+# Feature importance visualizer
+#
+# Author:  Nathan Danielsen <nathan.danielsen@gmail.com>
+# Created: Fri Mar 24 8:17:36 2018 -0500
+#
+# Copyright (C) 2018 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: missing.py [] nathan.danielsen@gmail.com.com $
+
+"""
+Implementation of missing values visualizers
+
+To Include:
+- Bar
+- Density Matrix (by time, specifiable index)
+- Heatmap
+
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from yellowbrick.utils import is_dataframe
+from yellowbrick.utils import is_structured_array
+from yellowbrick.features.base import DataVisualizer
+
+# from yellowbrick.style.colors import resolve_colors
+
+
+
+##########################################################################
+## Feature Visualizer
+##########################################################################
+
+
+class MissingValuesBarVisualizer(DataVisualizer):
+    """
+    """
+
+    def __init__(self,
+                 ax=None,
+                 x=None,
+                 y=None,
+                 features=None,
+                 classes=None,
+                 color=None,
+                 colormap=None,
+                 **kwargs):
+        """
+        """
+
+        super(MissingValuesBarVisualizer, self).__init__(ax, features, classes, color,
+                                                colormap, **kwargs)
+
+
+    def fit(self, X, y=None, **kwargs):
+        """
+        TODO if y, then color code the missing values in the chart?
+
+
+        """
+        nrows, ncols = df.shape
+
+        # Handle the feature names if they're None.
```

Comment:
I've actually investigated deeper how sklearn handles dataframes/structured arrays etc. It might not be relevant here, but I'm happy to go over it with you.